package/MDAnalysis/analysis/encore/dimensionality_reduction/stochasticproxembed.pyx

```diff
@@ -31,9 +31,11 @@ import logging
 import numpy
 cimport numpy
 
-cimport cstochasticproxembed
```

Comment:
Looks like a relative cimport (instead of this absolute one) would be correct here.

Reply:
good chance might be true. But replacing the pxd file with a single import is better and less code overall.

Reply:
@mtiberti @wouterboomsma , maybe just double-check that you are not missing anything with inlining the pxd file.

Reply:
ping @mtiberti  @wouterboomsma

Reply:
@kain88-de the ENCORE tests pass, don't they?

Let's give @mtiberti and @wouterboomsma another day but if there are no further comments then go ahead and merge.

Reply:
yep, looks good to me - actually cleaner